mne/tests/test_report.py

```diff
@@ -605,7 +605,6 @@ def test_scraper(tmpdir):
     rst = scraper(block, block_vars, gallery_conf)
```

Comment:
this line is the one that creates `op.join(app.builder.outdir, 'auto_examples')` (which then leads to a `FileExists` error on the deleted line)